sources/models/src/aws-k8s-1.30-nvidia/defaults.d/81-nvidia-k8s-container-toolkit.toml

```diff
@@ -0,0 +1 @@
+../../../shared-defaults/nvidia-k8s-container-toolkit.toml
```

Comment:
The PR is missing symlinks for other variants, we need symlinks for:

- aws-k8s-1.24-nvidia
- aws-k8s-1.25-nvidia
- aws-k8s-1.26-nvidia